include/alpaka/core/Common.hpp

```diff
@@ -37,6 +37,19 @@
 #    define ALPAKA_FN_HOST
 #endif
 
+//-----------------------------------------------------------------------------
+//! All functions marked with ALPAKA_FN_ACC or ALPAKA_FN_HOST_ACC that are exported to / imported from different
+//! translation units have to be attributed with ALPAKA_FN_EXTERN.
+//!
+//! Usage:
+//! ALPAKA_FN_ACC ALPAKA_FN_EXTERN auto add(std::int32_t a, std::int32_t b) -> std::int32_t;
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+#    define ALPAKA_FN_EXTERN SYCL_EXTERNAL
+#else
+#    define ALPAKA_FN_EXTERN
+#endif
+
+//-----------------------------------------------------------------------------
```

Comment:
Some time ago we removed all such comments. Please don't add new ones.

Reply:
Sorry, that must have been reintroduced by the squashing of commits.